cmd/manager/main.go

```diff
@@ -130,8 +131,16 @@ func main() {
 		os.Exit(1)
 	}
 
+	// Create and start a new auto detect process for this operator
+	autodetect, err := common.NewAutoDetect(mgr)
```

Comment:
not a big deal but seems putting this in ```common``` is not ideal.

Reply:
I can easily move it out. Any suggestions on where it should live?

Reply:
maybe ``crd ``` so it would become ```crd.NewAutoDetect```

Reply:
Most ```util ``` or ``` common ``` functions belong to a domain ``` crd.AutoDetect, images.Diff ``` etc so I would put them in packages that reflect their use if possible